Dockerfile

```diff
@@ -8,8 +9,27 @@ LABEL "com.github.actions.name"="Release Drafter"
 LABEL "com.github.actions.description"="Drafts your next release notes as pull requests are merged into master."
 LABEL "com.github.actions.icon"="edit-2"
 LABEL "com.github.actions.color"="orange"
+RUN node npm install -g pnpm@${PNPM_VERSION}
 WORKDIR /app
 COPY . .
-RUN yarn --frozen-lockfile
+RUN pnpm install --frozen-lockfile
+ENTRYPOINT [ "/app/node_modules/.bin/probot" ]
+CMD [ "receive", "/app/index.js" ]
+
+FROM node:16.16.0-alpine@sha256:9a2db0008b592cfde1b98aa8972a8051ccb80bb51dde8b06c3a2e3762794e89c as run
```

Comment:
The intent was to copy over only a single nodejs produced by [@vercel/ncc](https://github.com/vercel/ncc)